playbooks/repo-pip-setup.yml

```diff
@@ -0,0 +1,39 @@
+---
+# Copyright 2014, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# The purpose here is to allow for the environment to update/build the
+# python wheel files from the CURRENT release, as set in the rpc_release
+# variable.
+- name: Build new repo packages for a given release
+  hosts: hosts:all_containers
+  max_fail_percentage: 20
+  gather_facts: true
+  user: root
+  roles:
+    - pip_lock_down
+  post_tasks:
+    - name: Add new pip link
+      lineinfile:
+        dest: "{{ ansible_env.HOME }}/.pip/links.d/rpc_release.link"
+        line: "{{ openstack_repo_url }}/rpc-releases/{{ rpc_release }}"
```

Comment:
Small detail for consistency, but not functionally required - there should be a '/' at the end of this line just before the quotes